apps/storybook/docs/e_ontwerp/form/2_validation/form-validation.stories-doc.mdx

```diff
@@ -0,0 +1,75 @@
+import { Canvas } from '@storybook/blocks';
+import * as validationStories from './form-validation.stories';
+
+# Form - Validation
+> Uitgebreid voorbeeld van validatie met onze form componenten vind je op de [Form Demo pagina](/docs/ontwerp-form-demo--documentatie).
+
+Onze form componenten maken gebruik van [open-wc](https://open-wc.org/)'s [form participation](https://github.com/open-wc/form-participation) om form validatie te bekomen met web components die gebruik maken van native HTML form elements.
+
+Deze library is gebaseerd op de [native HTML form validatie](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation) en [Constraint Validation API](https://developer.mozilla.org/en-US/docs/Web/API/Constraint_validation) en poogt zo dicht mogelijk bij de native HTML form validatie te blijven.
+
+Daarnaast voorzien we ook de [`parseFormData()` helper functie](/docs/ontwerp-form-form-data--documentatie#parseformdataformelement-form-multiformcontrolnames-string) om met de [FormData API](https://developer.mozilla.org/en-US/docs/Web/API/FormData) om te gaan.
+
+## ValidityState in Formuliervalidatie
+
+Een essentieel onderdeel van de validatie is de [ValidityState interface](https://developer.mozilla.org/en-US/docs/Web/API/ValidityState), onderdeel van de native Validation API.
+Dit object geeft de huidige validatiestatus van een formuliercontrole weer en kan worden gebruikt om gedetailleerde foutmelding te verkrijgen.
+
+Die fout meldingen worden getoond door de [`vl-error-message-next` component](/docs/form-next-error-message--documentatie) die de `ValidityState` van de form control controleert.
+
+## Attributen van ValidityState
+
+Hieronder een overzicht van de gebruikte attributen van [ValidityState](https://developer.mozilla.org/en-US/docs/Web/API/ValidityState#instance_properties) met hun betekenis en bijhorende [validation attributen](https://developer.mozilla.org/en-US/docs/Web/HTML/Constraint_validation#validation-related_attributes):
+
+- `customError`: als je custom validatie instelt, [zie hier](/docs/ontwerp-form-custom-validation--documentatie) hoe je zelf een custom validator schrijft.
+- `patternMismatch`: als de waarde niet overeenkomt met het patroon dat is ingesteld door het `pattern` attribuut.
+- `rangeOverflow`: als de waarde hoger is dan de waarde die is gespecificeerd door het `max` attribuut.
+- `rangeUnderflow`: als de waarde lager is dan de waarde die is gespecificeerd door het `min` attribuut.
+- `tooLong`: als de waarde langer is dan de maximale lengte die is gespecificeerd door het `maxlength` attribuut.
+- `tooShort`: als de waarde korter is dan de minimale lengte die is gespecificeerd door het `minlength` attribuut.
+- `valueMissing`: als er een waarde is vereist, maar het invoerveld leeg is. Gespecifiëerd door het `required` attribuut.
```

Comment:
Gespecificeerd ipv Gespecifiëerd

Reply:
Had dit effectief opgezocht omdat ik hier over twijfelde :) en ik zie dat ik hier de 2 varianten heb gebruikt.

`gespecificeerd` lijkt het meest gebruikte in Nederland maar in Vlaanderen worden beide vormen gebruikt.

https://www.vlaanderen.be/team-taaladvies/taaladviezen/specifieren-specificeren

https://www.vlaamswoordenboek.be/definities/term/specifi%C3%ABren

Ga momenteel doen zoals het in de rest van storybook is gebruikt als dat goed is, namelijk `gespecifieerd`.